lib/css/syntax-highlighting/highlightjs.less

```diff
@@ -0,0 +1,85 @@
+@import "../exports/_stacks-exports";
+
+// reference doc https://highlightjs.readthedocs.io/en/latest/style-guide.html
+
+pre.hljs,
+code {
+    font-size: @fs-body1;
+    font-family: @ff-mono;
+    line-height: @lh-lg;
+    color: var(--highlight-color);
+    background-color: var(--highlight-bg);
+
+    .hljs-subst {
+        color: var(--highlight-color);
+    }
+
+    .hljs-meta,
+    .hljs-comment {
+        color: var(--highlight-comment);
+    }
+
+    .hljs-keyword,
+    .hljs-selector-tag,
+    .hljs-meta-keyword,
+    .hljs-doctag,
+    .hljs-section {
+        color: var(--highlight-keyword);
+        font-weight: bold;
+    }
+
+    .hljs-attr {
+        color: var(--highlight-attribute);
+    }
+    
+    
+    .hljs-attribute {
+        color: var(--highlight-symbol);
+    }
+
+    .hljs-name,
+    .hljs-type,
+    .hljs-number,
+    .hljs-selector-id,
+    .hljs-selector-class,
+    .hljs-quote,
+    .hljs-template-tag,
+    .hljs-deletion {
+        color: var(--highlight-namespace);
+    }
+
+    .hljs-string,
+    .hljs-regexp,
+    .hljs-symbol,
+    .hljs-variable,
+    .hljs-template-variable,
+    .hljs-link,
+    .hljs-selector-attr,
+    .hljs-selector-pseudo {
+        color: var(--highlight-variable);
+    }
+
+    .hljs-title,
+    .hljs-literal {
+        color: var(--highlight-literal);
+    }
+
+    .hljs-built_in,
+    .hljs-bullet,
+    .hljs-code,
+    .hljs-addition {
+        color: var(--highlight-punctuation);
+    }
+
+    .hljs-meta-string {
+        color: var(--highlight-variable);
+    }
+
+    .hljs-emphasis {
+        font-style: italic;
+    }
+
+    .hljs-strong {
+        font-weight: bold;
+    }
```

Comment:
For consistency, we probably shouldn't have bolding / italic in one highlighting and none in another. Considering just removing these.

Reply:
These classes specifically are required for markdown syntax highlighting:

![image](https://user-images.githubusercontent.com/14169651/82907685-ec4ff400-9f34-11ea-9abb-8a3c811b1789.png)

Reply:
I forgot to add these into the non-`highlightjs` themes since they aren't used there, but I can give it a go pretty easily.

Reply:
Removed the non-markdown `bold` styling for all themes